app/workers/package_manager_download_worker.rb

```diff
@@ -39,12 +39,7 @@ def initialize(platform, name, version)
     inqlude: PackageManager::Inqlude,
     julia: PackageManager::Julia,
     maven: PackageManager::Maven,
-    maven_atlassian: PackageManager::Maven::Atlassian,
```

Comment:
I _think_ this is only used when the worker is called with something like `maven_springlibs` and we aren't storing that value anywhere in the database. Does that sound right? I'm just trying to double check that removing these isn't going to break Project lookups for Maven projects that aren't in Central.

Reply:
AFAIK it's only a mapping from the depper-provided platforms to these subclasses in Libraries